lib/ansible/cli/doc.py

```diff
@@ -300,9 +300,9 @@ def get_plugin_metadata(plugin_type, plugin_name):
 
     @staticmethod
     def namespace_from_plugin_filepath(filepath, plugin_name, basedir):
-        if not basedir.endswith('/'):
-            basedir += '/'
-        rel_path = filepath.replace(basedir, '')
+        if not basedir.endswith(b'/'):
```

Comment:
Don't we usually subscribe to a "unicode sandwich" where functions pass around unicode strings and the function converts to bytes or native strings when it crosses an external boundary?